test/helpers.lua

```diff
@@ -288,11 +284,19 @@ module.uname = (function()
 end)()
 
 function module.is_os(s)
-  if not (s == 'win' or s == 'mac' or s == 'unix') then
+  if not (s == 'win'
+    or s == 'mac'
+    or s == 'unix'
+    or s == 'freebsd'
+    or s == 'openbsd'
+    or s == 'bsd') then
     error('unknown platform: '..tostring(s))
   end
-  return ((s == 'win' and module.iswin())
+  return ((s == 'win' and module.uname() == 'windows')
     or (s == 'mac' and module.uname() == 'darwin')
+    or (s == 'freebsd' and module.uname() == 'freebsd')
+    or (s == 'openbsd' and module.uname() == 'openbsd')
+    or (s == 'bsd' and string.find(module.uname(), 'bsd'))
     or (s == 'unix'))
```

Comment:
It seems that `is_os('unix')` is always true and is never used. Maybe it can be removed.